README.md

```diff
@@ -1,70 +1,61 @@
-
+- kvm as hypervisor for the VMs (you can check through the `kvm-ok` command)
 - [docker](https://www.docker.com/) >= 18.09 (for using kind and our deployment base image)
 - [docker-compose](https://docs.docker.com/compose/) >= 1.25.4 (for ease of use and for parallelizing control plane and partition deployment)
 - [kind](https://github.com/kubernetes-sigs/kind/releases) == v0.9.0 (for hosting the metal control plane on a kubernetes cluster v1.19.1)
-- [ovmf](https://wiki.ubuntu.com/UEFI/OVMF) to have a uefi firmware for virtual machines
-- the lab creates a virtual network 192.168.121.0/24 on your host machine, this hopefully does not overlap with other networks you have
+- [containerlab](https://containerlab.srlinux.dev/install/)
+- the lab creates a docker network on your host machine (`172.17.0.1`), this hopefully does not overlap with other networks you have
 - (recommended) haveged to have enough random entropy (only needed if the PXE process does not work)
 
 Here is some code that should help you setting up most of the requirements:
 
  ```bash
+# Install kvm
+sudo apt install -y git curl qemu qemu-kvm havaged
+
 # Install Docker
 curl -fsSL https://get.docker.com | sh
 # if you want to be on the safe side, follow the original installation
 # instructions at https://docs.docker.com/engine/install/ubuntu/
 
-# Install vagrant and other stuff
-wget https://releases.hashicorp.com/vagrant/2.2.14/vagrant_2.2.14_x86_64.deb
-sudo apt-get install ./vagrant_2.2.14_x86_64.deb qemu-kvm virt-manager ovmf net-tools libvirt-dev haveged
-
-# Ensure that your user is member of the group "libvirt" and "docker"
+# Ensure that your user is member of the group "docker"
 # you need to login again in order to make this change take effect
-sudo usermod -G libvirt,docker -a ${USER}
+sudo usermod -G docker -a ${USER}
 
-# Install libvirt plugin for vagrant
-vagrant plugin install vagrant-libvirt
+# Install containerlab
+bash -c "$(curl -sL https://get-clab.srlinux.dev)"
 
 # Install kind (kubernetes in docker), for more details see https://kind.sigs.k8s.io/docs/user/quick-start/#installation
-curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64
-chmod +x ./kind
-sudo mv ./kind /usr/local/bin/kind
+sudo curl -Lo /usr/local/bin/kind "https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64"
+sudo chmod +x /usr/local/bin/kind
 
 # Install docker-compose, for more details see https://docs.docker.com/compose/install/
-sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+sudo curl -Lo /usr/local/bin/docker-compose "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)"
 sudo chmod +x /usr/local/bin/docker-compose
 ```
 
-The following ports are getting used statically:
+The following ports are used statically on your host machine:
```

Comment:
Do we still use all these ports?

Reply:
Yes... but maybe we can get rid off it with the proposal I made here: https://github.com/metal-stack/mini-lab/issues/80#issuecomment-968623041